app/helpers/application_helper.rb

```diff
@@ -1,2 +1,26 @@
 module ApplicationHelper
+	
+	def friendly_age(birthdate)
+		years_old = Date.today - birthdate 
+		years_and_months = years_old.divmod(365)
+		years = years_and_months[0]
+		months = years_and_months[1] / 30
```

Comment:
Why not combine `months` and `months_i` below by just doing `months = (years_and_months[1] / 30).to_i` ? @YingniY